src/item/ItemFactory.php

```diff
@@ -98,6 +98,7 @@ public function __construct(){
 		$this->register(new GoldenApple(new IID(Ids::GOLDEN_APPLE), "Golden Apple"));
 		$this->register(new GoldenAppleEnchanted(new IID(Ids::ENCHANTED_GOLDEN_APPLE), "Enchanted Golden Apple"));
 		$this->register(new GoldenCarrot(new IID(Ids::GOLDEN_CARROT), "Golden Carrot"));
+		$this->register(new Spyglass(new IID(Ids::SPYGLASS), "Spyglass"));
```

Comment:
Not clear why this is up here...

Reply:
Because of the alphabetical order of class names

Reply:
This isn't in alphabetical order. S comes further down the alphabet.

Reply:
Oh, I realized my mistake. I'll fix it now